src/lib/plugins/validation/rule.component.ts

```diff
@@ -40,6 +40,7 @@ export class RuleComponent implements OnChanges {
 	@Input('isoDate') iso_date?: string;
 	@Input('equalToField') equal_to_field?: string;
 	@Input('listOf') list_of?: string;
+	@Input('customValidation') custom_validation?;
```

Comment:
Consider to rename it to `test|assert` please.

Reply:
renamed to 'test'